src/ensembl/src/shared/components/visibility-icon/VisibilityIcon.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import ImageButton from 'src/shared/components/image-button/ImageButton';
+
+import { ReactComponent as Eye } from 'static/img/track-panel/eye.svg';
+import { ReactComponent as EyePartial } from 'static/img/track-panel/eye-half.svg';
+
+import { Status } from 'src/shared/types/status';
+
+import styles from './VisibilityIcon.scss';
+
+type VisibilityIconStatus =
+  | Status.DEFAULT
```

Comment:
Should the visibility icon even have the `default` status? Isn't is just selected-unselected-partiallySelected-disabled?

Reply:
True. Its the question of using either default or unselected. Chose unselected. :-)

Reply:
I think we need the DEFAULT as we use it when PARTIALLY_SELECTED is passed as props - use it together with className passed to ImageButton component